doc/release/release_0.14.rst

```diff
@@ -0,0 +1,237 @@
+
+This is the last major release with an official support of Python 2, future
+releases will be developed broadly using Python 3 capabilities, and will be
+tested against Python 3 (or greater) environments only.
+
+This release is also a long-time support (LTS) release, which means that it will
+receive important backports and bugfixes for several years.
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+http://scikit-image.org
+
+
+New Features
+------------
+- Manual segmentation with matplotlib:
+  ``skimage.future.manual_polygon_segmentation``,
+  ``skimage.future.manual_lasso_segmentation``. (#2584)
+- Hysteresis thresholding:
+  ``skimage.filters.apply_hysteresis_threshold``. (#2665)
+- Lookfor function: ``skimage.lookfor``. (#2713)
+- Reworked montage function: ``skimage.util.montage``. (#2626)
+- 2D and 3D segmentation with morphological snakes:
+  ``skimage.segmentation.morphological_chan_vese``,
+  ``skimage.segmentation.morphological_geodesic_active_contour``. (#2791)
+- nD support for image moments:
+  ``skimage.measure.moments_central``, ``skimage.measure.moments_central``,
+  ``skimage.measure.moments_normalized``, ``skimage.measure.moments_hu``. (#2603)
+- Inertia tensor and its eigenvalues can now be computed outside of
+  regionprops; available in ``skimage.measure.inertia_tensor``. (#2603)
+- Cycle-spinning function for approximating shift-invariance by averaging
+  results from a series of spatial shifts:
+  ``skimage.restoration.cycle_spin``. (#2647)
+- Data generation with random_shapes function:
+  ``skimage.draw.random_shapes``. (#2773)
+- Haar-like feature: ``skimage.feature.haar_like_feature``,
+  ``skimage.feature.haar_like_feature_coord``,
+  ``skimage.feature.draw_haar_like_feature``. (#2848)
+- Subset of LFW database: ``skimage.data.cbcl_face_database``. (#2905)
+- Image moments from coordinate input: ``skimage.measure.moments_coords``,
+  ``skimage.measure.moments_coords_central``. (#2859)
+
+
+Improvements
+------------
+- ``VisuShrink`` method for ``skimage.restoration.denoise_wavelet``. (#2470)
+- ``skimage.transform.resize`` and ``skimage.transform.rescale`` have a new
+  ``anti_aliasing`` option to avoid aliasing artifacts when down-sampling
+  images. (#2802)
+- Support for multichannel images for ``skimage.feature.hog``. (#2870)
+- Non-local means denoising (``skimage.restoration.denoise_nl_means``) has
+  a new optional parameter, ``sigma``, that can be used to specify the noise
+  standard deviation. This enables noise-robust patch distance estimation. (#2890)
+- New ``alignment`` parameter in ``skimage.feature.plot_matches``. (#2955)
+
+
+API Changes
+-----------
+- ``skimage.util.montage.montage2d`` is now available as
+  ``skimage.util.montage2d``.
```

Comment:
Is this `montage` or `montage2d`? Note that the deprecation note below uses `montage` but this says `montage2d`.

Reply:
@jni Everything is correct there. `skimage.util.montage.` namespace has been removed, and `montage2d` function has been deprecated and moved to `skimage.util.`. On top of that, we introduced new function `skimage.util.montage` (fully reworked, with a better padding behaviour).
I've made some clarifications to the document.